frontend/src/app/libs/login/login.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { UserCredentials } from '../auth/models/userCredentials';
+import { AuthFacadeService } from '../auth/store/auth/auth.facade';
+
+@Component({
+	templateUrl: './login.component.html',
+	styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
+	public credentials: UserCredentials = new UserCredentials();
+	public errorMessageSub: Subscription;
+	public isAuthenticatedSub: Subscription;
+	public errorMessage: string;
+	public isAuthenticated: boolean;
+
+	constructor(private authFacadeService: AuthFacadeService) { }
+
+	public ngOnInit(): void {
+		this.errorMessageSub = this.authFacadeService.errorMessage$.subscribe(
```

Comment:
implement OnDestroy interface, create `destroy` subject and use `takeUntil` pipe to unsubscribe from subscriptions before component removed. So no need to store subscriptions in variables.